capabilities.js

```diff
@@ -4,6 +4,19 @@ define('capabilities', ['settings'], function(settings) {
         return !!m && m.matches;
     }
 
+    function detectOS() {
+        var macInfo = /^[^\(]+\(Macintosh.+Mac OS X 10[_\.](\d+)[^\)]+\)/
+            .exec(navigator.userAgent);
+        if (macInfo) {
+            return {
+                name: 'Mac OS X',
+                version: [10, parseInt(macInfo[1])],
```

Comment:
Chrome and Safari use underscores like `10_9_5` so it would need to manual tweaking. I think it makes sense to have it as an array of version numbers so I'm happy with this.